main/zoe/guide/working_with_ZCFSeats.md

```diff
@@ -0,0 +1,229 @@
+exited, `false` if it's still active. 
+
+- `ZCFSeat.exit(completion)` returns nothing. It is called when
+   a seat successfully finishes processing an offer, including making any
+   payouts. The seat is destroyed, and its object can no longer
+   interact with its previously associated contract instance.
+
+   The `completion` argument is usually a string, but can be any
+   object. It's only used for the notification sent to the contract
+   instance's `done()` function. `exit()` does not affect any other
+   open seats, outstanding promises, or the contract instance itself. 
+   
+- `ZCFSeat.fail(msg)` also returns nothing. It is called when an error
+   happened when the seat tried to process its offer. The `msg` argument
+   should be an appropriate error message for whatever happened. While the
+   contract instance still gets its current allocation, the seat is
+   destroyed and can no longer interact with the contract
+   instance. `fail()` does not affect any other open seats,
+   outstanding promises, or the contract instance itself. 
+
+   Agoric recommends using `fail()` to exit a seat with an error as
+   follows: 
+   ```js
+   throw seat.fail(Error('Something's wrong'));
+   ```
+   
+### Examining an offer and its contract instance
+   
+A `ZCFSeat` can get that seat's: 
+- Offer proposal. 
+- Current statged allocation
+- Amount allocated for one keyword. 
+- Current allocation for all keywords.
+   
+A `ProposalRecord` consists of the following three items,
+fully defined in the [`E(zoe).offer()` documentation](/zoe/api/zoe.html#e-zoe-offer-invitation-proposal-paymentkeywordrecord).
+For more
+information about proposals, see elsewhere
+in the [Zoe Guide](/zoe/guide/proposal.md#proposals). 
+- `want`: a record with keywords as keys and amounts as values.
+- `give`: a record with keywords as keys and amounts as values.
+- `exit`: the conditions under which the seat/offer can exit.
+
+Allocations represent the amounts to be paid out to each seat for
+a contract instance offer on exit, whether successful or not.
+Normal reasons for exiting are the user requesting to exit or the contract
+explicitly choosing to close out the seat. The guarantees also hold
+if the contract encounters an error or misbehaves. There are several
+methods for finding out what amount a current allocation is. 
+
+- `ZCFSeat.getProposal()` returns a `ProposalRecord` representing the
+   seat's associated offer proposal. This specifies the escrow
+   rules defining what the user who made the offer expects to get back
+   from it via Zoe. 
+   
+   Example:
+   `const { want, give, exit } = sellerSeat.getProposal();`
+
+- `ZCFSeat.getStagedAllocation()` returns the seat's current
+   statged allocation. This may change, as offers can be altered
+   in response to how they are received. For example,
+   if an offer of 10 Quatloos is rejected, a prospective buyer
+   may want to increase their offer to 15 Quatloos. 
```

Comment:
It can change from the contract reallocating, but the user on the other side can't add to or subtract from the balance.